### PR TITLE
Added Google colab

### PR DESCRIPTION
**Pull request by Lakhankumawat**
Instead of **.py** files i added **.ipynb** files so that a _newcomer feels ease_ with this repo and quizes are  easy to read .
Also added a bit of information in **README.md** and **designed** it to look nicerr

**PLEASE MERGE I M BIG FAN OF YOURS**  🙂

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,21 @@
-# Python-Quiz
+# Python-Quiz<img align="right"  src="https://img.icons8.com/color/96/000000/python.png" alt="py" /><br>
+<b>Test your knowledge with this python quiz! Determine the output of each function.</b> <br><br>
+Exercises for Python beginners
 
-Test your knowledge with this python quiz! Determine the output of each function.
+* [Python beginner exercises](https://pythonbasics.org/Exercises/)
```
Comment (techwithtim) [inline README.md]: remove all of this please

File: answers.py [hunk 2]
```diff
@@ -1,74 +0,0 @@
-# Q1
```
Comment (techwithtim) [inline answers.py:1]: add this file back, we should have both .ipynb and .py

File: pythonQuiz.py [hunk 3]
```diff
@@ -1,150 +0,0 @@
-def q1():
```
Comment (techwithtim) [inline pythonQuiz.py:1]: add this file back

**Review by techwithtim — CHANGES_REQUESTED** [review]
Please add the original two files back to the repo and modify the readme as commented. Thanks for the help!

**Comment by Lakhankumawat** [comment]
Okay 👍

**Pull request closed without merging**